assets/js/components/DashboardMainApp.js

```diff
@@ -298,7 +298,7 @@ export default function DashboardMainApp() {
 
 			{ showSurveyPortal && <CurrentSurveyPortal /> }
 
-			<MetricsSelectionPanel />
+			{ isKeyMetricsWidgetHidden === false && <MetricsSelectionPanel /> }
```

Comment:
Hi @ankitrox, I realise this follows the IB, but the IB doesn't fully cover the AC.

The `isKeyMetricsWidgetHidden` value will resolve to `false` even when the Key Metrics feature hasn't been set up which means as things stand we'll still be rendering the panel even when the feature has not been set up. We want to avoid this. _However_ there's an exception to this, which is that we _do_ want to render the panel when the Setup CTA is visible so the panel is ready to slide out if the user click on the **Select your own metrics** button:

![image](https://github.com/user-attachments/assets/e913ab7e-20d8-4c8e-bddd-022ca47e7670)

What this boils down to is we should retrieve a value as follows toward the top of this component:

```js
	const showKeyMetricsSelectionPanel = useSelect(
		( select ) =>
			// Show the selection panel if the Key Metrics feature is set up and is not hidden...
			( select( CORE_SITE ).isKeyMetricsSetupCompleted() === true &&
				isKeyMetricsWidgetHidden === false ) ||
			// ...or if the setup CTA is visible.
			( select( CORE_USER ).isAuthenticated() &&
				select( CORE_SITE ).isKeyMetricsSetupCompleted() === false &&
				select( CORE_USER ).isItemDismissed(
					KEY_METRICS_SETUP_CTA_WIDGET_SLUG
				) === false )
	);
```

Then update this line to be:

```js
			{ showKeyMetricsSelectionPanel && <MetricsSelectionPanel /> }
```

Please also update the QAB accordingly.